src/main/java/picard/fingerprint/CrosscheckFingerprints.java

```diff
@@ -646,20 +681,31 @@ private int checkFingerprintsBySample(final Map<FingerprintIdDetails, Fingerprin
             final FingerprintIdDetails lhsID = sampleToDetail1.get(sample);
             final FingerprintIdDetails rhsID = sampleToDetail2.get(sample);
 
+            final Fingerprint lhsFP = fingerprints1BySample.get(lhsID);
+            final Fingerprint rhsFP = fingerprints2BySample.get(rhsID);
```

Comment:
move these lines to below the if statement, so that it will be closer to where it is used.

Reply:
done